pkg/api/v1alpha1/authenticated_user.go

```diff
@@ -520,6 +527,27 @@ func (r *Router) updateAuthenticatedUser(c *gin.Context) {
 		return
 	}
 
+	updateNotificationPublishEventErr := error(nil)
```

Comment:
```suggestion
	var updateNotificationPublishEventErr error
```

is this your intention here?